src/main/java/com/commercetools/project/sync/product/ProductSyncer.java

```diff
@@ -115,10 +122,54 @@ protected CompletionStage<List<ProductDraft>> transform(@Nonnull List<ProductPro
                 }
                 return Collections.emptyList();
               }
-              return productDrafts;
+              return removeDiscountedFromPrices(productDrafts);
             });
   }
 
+  /**
+   * Currently java-sync does not support discounted price sync. This workaround is to remove
+   * discounted prices from syncing.
+   *
+   * <p>Issue: https://github.com/commercetools/commercetools-project-sync/issues/363
+   */
+  private List<ProductDraft> removeDiscountedFromPrices(List<ProductDraft> productDrafts) {
+    return productDrafts
+        .stream()
+        .map(
+            productDraft -> {
+              List<ProductVariantDraft> productVariants =
+                  productDraft
+                      .getVariants()
+                      .stream()
+                      .map(this::createProductVariantDraftWithoutDiscounted)
+                      .collect(Collectors.toList());
+              ProductVariantDraft masterVariant = productDraft.getMasterVariant();
```

Comment:
It can be better like this:

final ProductVariantDraft masterVariant = productDraft.getMasterVariant();
final ProductVariantDraft masterVariantDraft =
                  (masterVariant != null) ? createProductVariantDraftWithoutDiscounted(masterVariant) : null;